test/Serialization/modularization-error.swift

```diff
@@ -18,8 +18,18 @@
 // RUN: %target-swift-frontend -emit-sil %t/LibWithXRef.swiftmodule -module-name LibWithXRef -I %t \
 // RUN:   -experimental-force-workaround-broken-modules 2>&1 \
 // RUN:   | %FileCheck --check-prefixes CHECK-WORKAROUND %s
-// CHECK-WORKAROUND: warning: attempting forced recovery enabled by -experimental-force-workaround-broken-modules
-// CHECK-WORKAROUND: note: reference to type 'MyType' broken by a context change; 'MyType' was expected to be in 'A', but now a candidate is found only in 'B'
+// CHECK-WORKAROUND: warning: reference to type 'MyType' broken by a context change; 'MyType' was expected to be in 'A', but now a candidate is found only in 'B'
+// CHECK-WORKAROUND: A.MyType
+// CHECK-WORKAROUND: ^
+// CHECK-WORKAROUND: A.swiftmodule:1:1: note: declaration was expected to be found in this module
+// CHECK-WORKAROUND: <binary format>
+// CHECK-WORKAROUND: ^
+// CHECK-WORKAROUND: B.swiftmodule:1:1: note: declaration was found in this module
+// CHECK-WORKAROUND: <binary format>
+// CHECK-WORKAROUND: ^
+// CHECK-WORKAROUND: LibWithXRef.swiftmodule:1:1: note: attempting forced recovery enabled by -experimental-force-workaround-broken-modules
+// CHECK-WORKAROUND: <binary format>
+// CHECK-WORKAROUND: ^
```

Comment:
@bnbarham I inverted the order back as we discussed. With the warning now showing the synthesized code I think this is more informative.